.github/workflows/scorecards-analysis.yml

```diff
@@ -9,30 +9,36 @@ on:
 permissions: read-all
 
 jobs:
-  analysis:
-    name: Scorecards analysis
+  setup:
+    name: Scorecards analysis setup
```

Comment:
Question: Is this piece related to scorecards? It looks like the script parses the DEPS file but does scorecards utilize that parsed file?

Reply:
It does parse the DEPS file and creates the flattened dependency file that I am using in vulnerability scanning. 
The deps scanning action has already been running in preparation for the vulnerability scanning which will happen in the steps afterwards. 
I'm not changing any of that with these changes, just refactoring

Reply:
That makes sense. My question was pointed toward whether this belongs in the scorecards action at all or if it would better fit in its own vulnerability scanning action.

Reply:
Ah ok I understand. I suppose we could make a new job altogether that for right now has just this deps_parser step in it. You are right, it isn't necessarily related to scorecards, it is setup step for the later vuln scanning